### DISCUSSION TITLE
part-ii/2015-123/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#177] [Question comments]
Part II 2015 Riemann Surfaces: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2015-123/

**Comment by silkca** [comment]
The first part of the question is bookwork.

The second part relies on the fact that if $f=g \circ h$, then $\deg f = \deg g + \deg h$, and the degree of $Q$ in terms of the degrees of $p$ and $q$ is standard knowledge. The third part is quite simple given the second part.
